seed_identity_store/settings.py

```diff
@@ -150,7 +162,7 @@
 CELERY_CREATE_MISSING_QUEUES = True
 
 CELERY_TASK_SERIALIZER = 'json'
-CELERY_RESULT_SERIALIZER = 'json'
+CELERY_RESULT_SERIALIZER = 'pickle'
```

Comment:
@KaitCrawford @hodgestar yes, it's caused by this unfortunate line that passes two model instances around (which are then unused by us) https://github.com/zapier/django-rest-hooks/blob/master/rest_hooks/models.py#L88 - I'm going to put together a PR for the project